test/attr/Inputs/DeprecatedSymbolsModule.swift

```diff
@@ -1,3 +1,4 @@
+// RUN: %target-typecheck-verify-swift
```

Comment:
We don't usually add `RUN` lines to input files (I'm not even sure if lit evaluates them). Note the file will be typechecked anyway when emitting it as a module in the main test.

Reply:
It seems to be needed here. See lines 33 where, in [this](https://github.com/apple/swift/pull/59232/commits/630be7fc13052b946e90f0738533e65480a831e8) commit, it was supposed to be aware that the 2 warnings were going to appear, however, the test failures indicate that it wasn't aware that those 2 warnings were going to occur.

Reply:
Ah in that case you'll want to add `-verify` to the `RUN` line that does the `-emit-module` in the main test file, rather than add the line here.